source/help/getting-started/signing-in.rst

```diff
@@ -4,8 +4,9 @@ Signing in
 --------------
 
 You can sign in to your team from the root page of your Mattermost site
-``https://domain.com``. After signing in, you will be able to view a
-list of your teams and select which one to open.
+``https://domain.com``. After signing in, the team that appears first on your
```

Comment:
Thinking about it some more, if you do something like http://<domain>.com, users still need to know what to replace <domain> with, and they get that from the administrator or by some other means. So they pretty much already know where the sign-in page is, and if they don't, then the doc should help them with that. So maybe  you could say something like:

*To sign in, navigate to the Mattermost sign-in page. You can get the URL of the sign-in page from your administrator. After you sign in, you can view a list of your teams and select which one to open.*